Package.swift

```diff
@@ -9,6 +9,10 @@ let package = Package(
             name: "Backtrace",
             targets: ["Backtrace"]
         ),
+        .executable(
```

Comment:
I would remove the explicit `executable` target here. If you leave it, then it will show up _everywhere_. Ie say you add `https://github.com/vapor/vapor` in Xcode, Xcode's UI will not allow you to add a dependency on `Sample`.

For `swift run Sample` to work, you do _not_ need to add the `products: [.executable(...)]`. The products are merely to export it to other packages such that they can depend on it.

Reply:
Ah, SGTM